_projects/add-repo-license.md

```diff
@@ -49,7 +49,7 @@ Start by looking through
 sorted by "most stars" or "most forks".
 
 Check that GitHub does not show a recognised license at the top of the repository.
-See [Adding a license](https://help.github.com/articles/adding-a-license-to-a-repository/)
+See [Adding a license](https://help.github.com/articles/adding-a-license-to-a-repository)
```

Comment:
im not able to reproduce an error which caused this change, or other removals of trailing `/`.  maybe it was transient.